src/shared/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+class Utils {
+
+    static formatDate(dateStr: string): string {
+        const date = new Date(dateStr);
+
+        return moment(date).format("YYYY MM DD");
+    }
+
+    static formatGender(gender: string): string {
+        return gender.includes('F') ? 'Female' : 'Male';
+    }
+
+    static calculatePatientAge(patientDOB: string): number {
+        const today = new Date();
+        const dob = new Date(patientDOB);
+        return today.getFullYear() - dob.getFullYear();
+    }
+}
```

Comment:
I was wondering what was the reason behind having a Class with static methods?

Personally, using a Class with static methods gives me a "code smell" since I don't see a reason to have a class here. Instead, if I were you, I'd just export functions.

@janakitti what is your opinion about this?
@colinywu if you do end up making them into exported functions, only do so if time permits.

Reply:
Not necessarily think it's a bad design practice, they behave very similarly, however, one has the benefit of tree-shaking.